examples/tests/sdk_tests/std/map.test.w

```diff
@@ -15,3 +15,210 @@ let mvalues = m.values();
+
+
+let trilogies = { "PT" => 2, "OT" => 1, "ST" => 3 };
+assert(trilogies.size() == 3);
+
+let mutTrilogies = MutMap<num> { "PT" => 2, "OT" => 1, "ST" => 3 };
+assert(mutTrilogies.size() == 3);
+mutTrilogies.delete("ST"); // One could argue that it's better if this does not exist
+assert(mutTrilogies.size() == 2);
+
+test "size()" {
+    let trilogies = { "PT" => 2, "OT" => 1, "ST" => 3 };
+    assert(trilogies.size() == 3);
+
+    let mutTrilogies = MutMap<num> { "PT" => 2, "OT" => 1, "ST" => 3 };
+    assert(mutTrilogies.size() == 3);
+    mutTrilogies.delete("ST");
+    assert(mutTrilogies.size() == 2);
+}
+
+
+let greeting = { "hello" => "there!" };
+assert(greeting.get("hello") == "there!");
+
+let general: str? = greeting.get("grievous");
+assert(general == nil);
+
+let mutGreeting = MutMap<str>{ "general" => "kenobi" };
+assert(mutGreeting.get("general") == "kenobi");
+
+let Viceroy: str? = mutGreeting.get("gunray");
+assert(Viceroy == nil);
+
+test "get()" {
+    let greeting = { "hello" => "there!" };
+    assert(greeting.get("hello") == "there!");
+
+    let general: str? = greeting.get("grievous");
+    assert(general == nil);
+
+    let mutGreeting = MutMap<str>{ "general" => "kenobi" };
+    assert(mutGreeting.get("general") == "kenobi");
+
+    let Viceroy: str? = mutGreeting.get("gunray");
+    assert(Viceroy == nil);
+}
+
+
+let legion501 = { "fives" => "CT-5555", "rex" => "CT-7567" , "appo" => "CC-1119", "jesse" => "CT-5597" };
+assert(legion501.has("fives") == true);
+assert(legion501.has("rex") == legion501.has("jesse"));
+assert(legion501.has("cody") == false);
+
+let padawans = MutMap<str> { "ahsoka" => "anakin", "anakin" => "kenobi", "kenobi" => "qui gon", "qui gon" => "dooku", "dooku" => "yoda" };
+assert(padawans.has("anakin") == true);
+assert(padawans.has("qui gon") == padawans.has("dooku") == padawans.has("anakin"));
+assert(padawans.has("windu") == padawans.has("sifo dyas"));
+assert(padawans.has("revan") == false);
+
+test "has()" {
+    let legion501 = { "fives" => "CT-5555", "rex" => "CT-7567" , "appo" => "CC-1119", "jesse" => "CT-5597" };
+    assert(legion501.has("fives") == true);
+    assert(legion501.has("rex") == legion501.has("jesse"));
+    assert(legion501.has("cody") == false);
+
+    let padawans = MutMap<str> { "ahsoka" => "anakin", "anakin" => "kenobi", "kenobi" => "qui gon", "qui gon" => "dooku", "dooku" => "yoda" };
+    assert(padawans.has("anakin") == true);
+    assert(padawans.has("qui gon") == padawans.has("dooku") == padawans.has("anakin"));
+    assert(padawans.has("windu") == padawans.has("sifo dyas"));
+    assert(padawans.has("revan") == false);
+}
+
+
+let forceUsers = { "sith" => ["malak", "vader", "bane"], "jedi" => ["sunrider", "bastila", "bindo"]};
+let userKeys = forceUsers.values();
+let valArr = [["malak", "vader", "bane"], ["sunrider", "bastila", "bindo"]];
+assert(userKeys.length == valArr.length);
+for i in 0..userKeys.length {
+    assert(userKeys.at(i) == valArr.at(i));
+}
```

Comment:
it will compare two arrays of size 3, right?

Reply:
Yeah it would. I originally wanted to do a direct assertion but that fails for some unknown reasons as I mentioned in the description.

Reply:
cool! can you please add a link to the bug?

Reply:
It's already linked in the description, but [here](https://github.com/winglang/wing/issues/4118) it is. I guess I'll also add a direct reference to the bug number in the description.

Reply:
I'm sorry, I meant as a comment in the code (so the next one to read/ copy this code will be aware of the issue)